lib/gli/monkeypatches/with_indifferent_access.rb

```diff
@@ -0,0 +1,17 @@
+# adapted from https://gist.github.com/Integralist/9503099
+class Object
```

Comment:
Looking over the rest of the PR, I think a better way to do this is make a class whose job is to load the configuration and have it do all this string/symbol copying, so we don't infect every object in the entire system with a method that's only needed in a couple of places.